src/main/java/com/amihaiemil/versioneye/Me.java

```diff
@@ -57,4 +59,10 @@
      * @return Favorites
      */
     Favorites favorites();
+    
+    /**
+     * Post authenticated user to the server.
+     * @param authenticated Authenticated user.
+     */
+    void post(JsonObject authenticated);
```

Comment:
@SherifWaly Well, post(...) method for authenticated user doesn't make much sense, because with ``RtVersionEye`` the user authenticates by simply providing the token in the ctor. So ``post(...)`` for RtVersionEye will forever throw UnsupportedOperationException. 

Instead, I think it makes more sense, for ``MkVersionEye`` to accept an ``MkUser`` in its ctor. This ``MkUser`` will wrap the json and provide meaningful methods to add attributes, like this:

```java
MkUser authenticated = new MkJsonUser();
authenticated = authenticated.username("SherifWaly");
authenticated = authenticated.fullName("Sherif Waly");
//...

VersionEye ve = new MkVersionEye(authenticated)
```
And inside, MkVersionEye will add the "authenticated" user to MkServer and keep its username to identify him later (each ``MkVersionEye`` instance should have only one authenticated user and know about it, while in ``MkServer``'s Json, more users could be authenticated at once (if you created more instances of MkVersionEye). Makes sense? :D 

But this of course is another task, add a puzzle for it, I would say 1h estimation.

For other cases, the post method will stay, but it should be fluent and don't ask the user for a JsonObject - there should be methods for each possible attribute. I mean, the whole scope of the lib is to be simple to use, the user should not have to consult the documentation to see the format of the objects in order to use it -- if they have to do that, they might as well make their own HTTP calls :D